packages/manager/apps/pci-rancher/src/components/Title/Title.tsx

```diff
@@ -0,0 +1,38 @@
+import {
+  ODS_THEME_COLOR_HUE,
+  ODS_THEME_COLOR_INTENT,
+  ODS_THEME_TYPOGRAPHY_LEVEL,
+  ODS_THEME_TYPOGRAPHY_SIZE,
+} from '@ovhcloud/ods-common-theming';
+import { OsdsText } from '@ovhcloud/ods-components/react';
+import React from 'react';
+
+const CommonTitle: React.FC<{
+  title: string;
+  typoSize: ODS_THEME_TYPOGRAPHY_SIZE;
+}> = ({ title, typoSize }) => {
+  return (
+    <OsdsText
+      level={ODS_THEME_TYPOGRAPHY_LEVEL.heading}
+      size={typoSize}
+      color={ODS_THEME_COLOR_INTENT.text}
+      hue={ODS_THEME_COLOR_HUE._800}
+    >
+      {title}
+    </OsdsText>
+  );
+};
+
+export const Title: React.FC<{ title: string }> = ({ title }) => {
```

Comment:
It would probably be more natural to put the string as a children instead of a prop named "title" for a typography component

Reply:
You can set the children type to `string` (even though I don't think it's a good idea to force anything on the developer because you can't foresee the future usecases we will face)

Reply:
Yes but we can't forsee how we will need to style around it